python/nx-cugraph/_nx_cugraph/__init__.py

```diff
@@ -31,14 +31,15 @@
         # BEGIN: functions
         "betweenness_centrality",
         "edge_betweenness_centrality",
+        "isolates",
         "louvain_communities",
         # END: functions
     },
     "extra_docstrings": {
         # BEGIN: extra_docstrings
         "betweenness_centrality": "`weight` parameter is not yet supported.",
         "edge_betweenness_centrality": "`weight` parameter is not yet supported.",
-        "louvain_communities": "`threshold` and `seed` parameters are currently ignored.",
+        "louvain_communities": "`seed` parameter is currently ignored.",
```

Comment:
Do we need to add something in the docs about `max_level` being capped at 500?

Reply:
Added `(max: 500)` to the max_level parameter doc.